### DISCUSSION TITLE
Add measurements to Splat

### DISCUSSION THREAD

**Discussion by Jonathan-Gore** [pierotofy/OpenSplat#70] [General]
I think I may be misunderstanding what Gaussian Splats exactly are.

But is there any way to add the ability to set reference dimensions so we could derive measurements from Gaussian Splat files?

**Comment by pfxuan** [comment]
To get the metric scale from 3DGS model, we can either recover it from the original SfM sparse point cloud before training starts or perform a downstream calibration after training completes.

**Reply by Jonathan-Gore** [reply]
Gotcha, both of those approaches make sense. 

How would one go about doing that in the context of an OpenSplat workflow. Is there a specific script or library that would be used, or software application?

**Reply by jared-krauss** [reply]
> Gotcha, both of those approaches make sense.
> 
> How would one go about doing that in the context of an OpenSplat workflow. Is there a specific script or library that would be used, or software application?

What did you figure out? Curious.
